usecases/schema/tenant.go

```diff
@@ -238,3 +238,32 @@ func (m *Manager) onDeleteTenants(ctx context.Context, class *models.Class, req
 
 	return nil
 }
+
+// GetTenants is used to get tenants of a class.
+//
+// Class must exist and has partitioning enabled
+func (m *Manager) GetTenants(ctx context.Context, principal *models.Principal, class string) ([]*models.Tenant, error) {
+	if err := m.Authorizer.Authorize(principal, "get", tenantsPath); err != nil {
+		return nil, err
+	}
+	// validation
+	cls := m.getClassByName(class)
+	if cls == nil {
+		return nil, fmt.Errorf("class %q: %w", class, ErrNotFound)
+	}
+	if !schema.MultiTenancyEnabled(cls) {
+		return nil, fmt.Errorf("multi-tenancy is not enabled for class %q", class)
+	}
+
+	m.shardingStateLock.Lock()
+	var tenants []*models.Tenant
+	if ss := m.state.ShardingState[cls.Class]; ss != nil {
+		tenants = make([]*models.Tenant, len(ss.Physical))
+		for tenant := range ss.Physical {
+			tenants = append(tenants, &models.Tenant{Name: tenant})
```

Comment:
when you create a slice with a fixed size for example 4, then first 4 entries are empty, and you put those entries there with accessing the slice with index like `arrar[i] = value`, but when you use `append` it then adds more data to already created slice you will end up with 4+ lengith

Reply:
juuuust noticed

Reply:
In this case you should create either create array with:
```
tenants := []*models.Tenant{}
```
or
```
tenants = make([]*models.Tenant, 0)
```